packages/console/src/components/Switch/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { forwardRef, HTMLProps, ReactNode } from 'react';
+
+import * as styles from './index.module.scss';
+
+type Props = HTMLProps<HTMLInputElement> & {
+  // eslint-disable-next-line react/require-default-props
```

Comment:
Avoid using `eslint-disable` by wrapping `forwardRef` at the end of the file.

```typescript
export default forwardRef<HTMLInputElement, Props>(Switch);
```

Reply:
It doesn't work, also, many of our components that use native HTMLElement have the same issue.

Reply:
It does work. Check our `Card` component for reference

Reply:
updated